pkg/quota/set_diskquota.go

```
@@ -37,13 +38,15 @@ func processSetQuotaReexec() {
 		}
 	}()
 
-	if len(os.Args) != 3 {
-		err = fmt.Errorf("invalid arguments: %v, it should be: %s: <path> <size>", os.Args, os.Args[0])
+	if len(os.Args) != 4 {
+		err = fmt.Errorf("invalid arguments: %v, it should be: %s: <path> <size> <quota id>", os.Args, os.Args[0])
 		return
 	}
 
 	basefs := os.Args[1]
 	size := os.Args[2]
+	id, _ := strconv.Atoi(os.Args[3])
```

I am wondering if we should not ignore the error. Maybe we can just record it.